specs/Standards/Sould Bound Token/README.md

```diff
@@ -0,0 +1,456 @@
+}
+
+
+trait SBTRegistry {
+    /**********
+    * QUERIES
+    **********/
+
+    /// get the information about specific token ID issued by `ctr` SBT contract.
+    fn sbt(&self, ctr: AccountId, token: TokenId) -> Option<Token>;
+
+    /// returns total amount of tokens issued by `ctr` SBT contract.
+    fn sbt_supply(&self, ctr: AccountId) -> u64;
+
+    /// returns total amount of tokens of given kind minted by this contract
+    fn sbt_supply_by_kind(&self, ctr: AccountId, kind: KindId) -> u64;
+
+    /// returns total supply of SBTs for a given owner.
+    fn sbt_supply_by_owner(&self, ctr: AccountId, account: AccountId) -> u64;
+
+    /// returns true if the `account` has a token of a given `kind`.
+    fn sbt_supply_by_kind(&self, ctr: AccountId, account: AccountId, kind: KindId) -> bool;
+
+    /// Query sbt tokens issued by a given contract.
+    /// If `from_index` is not specified, then `from_index` should be assumed
+    /// to be the first valid token id.
+    fn sbt_tokens(
+        &self,
+        ctr: AccountId,
+        from_index: Option<u64>,
+        limit: Option<u32>,
+    ) -> Vec<TokenId>;
+
+    /// Query sbt tokens by owner
+    /// If `from_kind` is not specified, then `from_kind` should be assumed to be the first
+    /// valid kind id.
+    /// Returns list of pairs: `(Contract address, list of token IDs)`.
+    fn sbt_tokens_by_owner(
+        &self,
+        account: AccountId,
+        ctr: Option<AccountId>,
+        from_kind: Option<u64>,
+        limit: Option<u32>,
+    ) -> Vec<(AccountId, Vec<TokenId>)>;
+
+    /*************
+     * Transactions
+     *************/
+
+    /// Creates a new, unique token and assigns it to the `receiver`.
+    /// `token_spec` is a vector of pairs: owner AccountId and TokenMetadata.
+    /// each TokenMetadata must have non zero `kind`.
+    /// Must be called by a SBT contract.
+    /// Must emit `Mint` event.
+    /// Must provide enough NEAR to cover registry storage cost.
+    // #[payable]
+    fn sbt_mint(
+        &mut self,
+        token_spec: Vec<(AccountId, TokenMetadata)>,
+    ) -> Vec<TokenId>;
+
+    /// sbt_recover reassigns all tokens from the old owner to a new owner,
+    /// and registers `old_owner` to a burned addresses registry.
+    /// Must be called by a SBT contract.
+    /// Must emit `Recover` event.
+    /// Must be called by an operator.
+    /// Must provide enough NEAR to cover registry storage cost.
+    /// Requires attaching enough tokens to cover the storage growth.
+    // #[payable]
+    fn sbt_recover(&mut self, from: AccountId, to: AccountId);
+
+    /// sbt_renew will update the expire time of provided tokens.
+    /// `expires_at` is a unix timestamp (in seconds).
+    /// Must be called by a SBT contract.
+    /// Must emit `Renew` event.
+    fn sbt_renew(&mut self, tokens: Vec<TokenId>, expires_at: u64, memo: Option<String>);
+
+    /// Revokes SBT, could potentailly burn it or update the expire time.
```

Comment:
I just randomly noticed the type: "potentailly" -> "potentially"

Reply:
thanks, fixed few other typos as well.